sycl/test-e2e/Matrix/element_wise_all_sizes.cpp

```diff
@@ -5,11 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// RUN: %{build} -fsycl-device-code-split=off -o %t_split.out
```

Comment:
Pls, move the comment from the old test also, so that it is clearer, why we have that
```suggestion
// This is a version of element_wise_all_sizes test with disabled device code
// split to test against fixed bug in IGC
// RUN: %{build} -fsycl-device-code-split=off -o %t_split.out
```

Reply:
You only copied one line of text, it doesn't make sense without second line